### DISCUSSION TITLE
I need help with VSOL V2802RH

### DISCUSSION THREAD

**Discussion by DonuyThompson** [Anime4000/RTL960x#342] [General]
I need help with VSOL V2802RH
I want to delete LOID
LOID PASSWD
i.e. leave it empty. But it does not accept saving

![2024-12-07_173825](https://github.com/user-attachments/assets/f614990c-1c5f-44ab-85d0-6386f671892d)

**Comment by Anime4000** [comment]
tried to export config and edit that way?
